calicoctl/commands/build_info.go

```diff
@@ -15,8 +15,4 @@
 // WARNING: This file is overwritten by a script in the release build process.
 package commands
 
-const (
```

Comment:
I was thinking the same thing and they're used only in version.go. I'll combine them in version.go.